<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CpuState</class>
 <widget class="QWidget" name="CpuState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>233</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">margin: 5px;</string>
   </property>
   <property name="text">
    <string>CPU使用率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>35</y>
     <width>300</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:16px;
border:0.5px solid grey;
</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>251</y>
     <width>35</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>100</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
border:1px solid grey;
background-color:rgb(0, 0, 0);
background-image:url(:/qss/ledunlit.bmp);
background-repeat: repeat-y;
padding:2px;
position:center;
}

QProgressBar::chunk {
background:rgb(0, 255, 0);
height: 2px;
margin: 0.5px;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string notr="true">%p%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>35</y>
     <width>35</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
border:1px solid grey;
background-color:rgb(0, 0, 0);
background-image:url(:/qss/ledunlit.bmp);
background-repeat: repeat-y;
padding:2px;
position:center;
}

QProgressBar::chunk {
background:rgb(0, 255, 0);
height: 2px;
margin: 0.5px;
}

</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="format">
    <string notr="true">%p%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>449</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">margin: 5px;</string>
   </property>
   <property name="text">
    <string>内存</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>251</y>
     <width>300</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:16px;
border:0.5px solid grey;
</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>graphicsView</zorder>
  <zorder>label_2</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>progressBar_2</zorder>
  <zorder>progressBar_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
